### ISSUE TITLE
数据集划分问题

### ISSUE THREAD

**Issue by yuhua666** [ZiweiWangTHU/BiDet#45]
您在训练代码时，只用到了训练集和测试集，未使用验证集(训练过程没有评估结果)，是这样吗？

**Comment by Wuziyi616** [comment]
你好 感谢你对我们工作的兴趣
- 对于 PASCAL VOC, 我们训在 train+val set, 测试在 test set, 这是 detection 文章常用的做法
- 对于 COCO 2017, 因为 test set label 无法获得因此无法 report performance, 所以我们训在 training set, 测试在 val set, 这也是其他 detection 文章常用的做法

**Comment by yuhua666** [comment]
您好，感谢您的回复。
这样的话就没有验证集，训练的时候直接拿训练集当作验证集，是这样的吗？

**Comment by Wuziyi616** [comment]
是的 可以这么认为

**Comment by yuhua666** [comment]
感谢，我想采用跟您一样的数据集划分来训练SSD、YOLO模型，我直接将VOC train+val作为训练集，同时作为验证集，然后在2007 test set上进行测试，这样的划分方法是正确的吗？简单地说，将train和val选择同样的集合。

**Comment by Wuziyi616** [comment]
可以的

**Comment by yuhua666** [comment]
您好，想再次请教一下，您论文里展示的表1结果里的mAP，指的是mAP 0.5:0.95还是mAP 0.5？

**Comment by yuhua666** [comment]
我再次拜读了您的论文
文中提及“The COCO dataset consists of images from 80 different categories. We conducted experiments on the 2014 COCO object detection track. We trained our model with the combination of 80k images from the training set and 35k images sampled from validation set (trainval35k [2]) and tested our method on the remaining 5k images in the validation set (minival [2]).”
BiDet应该是在COCO 2014上跑的吧，且训练集、验证集、测试集图片数量分别为80k、35k、5k？

**Comment by Wuziyi616** [comment]
你好 我记不太清楚了，但 PASCAL VOC 上应该只有 mAP 0.5 没有 mAP 0.5:0.95? 报的应该是 mAP 0.5

COCO 2014 和 COCO 2017 其实是一样的，唯一区别是 2017 training set == 2014 training set + val set 中选出来的 35k subset (the so-called trainval35k), 而 2017 val set == 2014 val set 去除 35k subset 后剩下的 5k. 所以我的实际做法是 train on (80k+35k), test on 5k, 你可以理解成是在 COCO 2017 上进行了实验；我也建议你用 COCO 2017, 因为 2014 里面的 trainval35k 的文件似乎 link 失效了

**Comment by Wuziyi616** [comment]
所以仍然是没有验证集，训练的时候直接拿训练集当作验证集，和 PASCAL VOC 一样

**Comment by yuhua666** [comment]
明白了，感谢！
